fvtest/porttest/omrvmemTest.cpp

```diff
@@ -722,24 +722,37 @@ TEST(PortVmemTest, vmem_test_double_mapping)
 	size_t arrayletLeafSize = SIXTEEN_KB; // 16KB
 	char vals[ARRAYLET_COUNT] = {'3', '5', '6', '8', '9', '0', '1', '2'};
 	size_t totalArrayletSize = 0;
-	void* arrayletLeaveAddrs[ARRAYLET_COUNT];
+	/* In OpenJ9 we must malloc the leaves addresses, so we test it here as well */
+	uintptr_t *arrayletLeaveAddrs = (uintptr_t *)malloc(ARRAYLET_COUNT * sizeof(uintptr_t));
+	BOOLEAN shouldDealocateAddrs = TRUE;
```

Comment:
Do we need this variable? Can you not just check to see if arrayletLeavesAddrs is NULL?